documentation/test/test_get_code_snippets.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+from unittest.mock import patch
+from pathlib import Path
+
+from local_gen import get_code_snippets
+
+class TestGetCodeSnippets:
+    @pytest.fixture
+    def fake_test_dir(self):
+        return Path("/fake/test/dir")
+
+    @patch('local_gen.read_file_contents')
+    @patch('local_gen.glob.iglob')
```

Comment:
nit: keep quote usage consistent (you seems to be using double quote everywhere else and that’s also what tools like `black` suggests)
```suggestion
    @patch("local_gen.read_file_contents")
    @patch("local_gen.glob.iglob")
```